src/lib/helpers.js

```
@@ -46,7 +45,7 @@ export function createHelpers({ cache, logger, webAPI }) {
     async findOne(entity, id, params) {
       const url = `/${entity}/${id}`
       const resp = await webAPI.get(url, { params })
-      return (resp.data && resp.data.data) || []
+      return (resp.data && resp.data.data) || {}
```

Should be `return resp.data`. The findOne and findMany are fundamental methods for hitting the API. findOne should throw an error if not found (because the get will fail due to 404). findMany will always return an array.